src/test/components/ActiveTabTimeline.test.js

```diff
@@ -153,9 +153,9 @@ describe('ActiveTabTimeline', function() {
     });
 
     it('does not display the resources panel if there are no resource tracks', () => {
-      const { getState, queryByText } = setup();
+      const { getState } = setup();
```

Comment:
Strangely, in some cases where we use `setup`, the rule couldn't find the issue. I fixed a few of the ones where the rule couldn't find the problem, but not all. I expect that some future update will be able to find them.